htdocs/.htaccess

```diff
@@ -15,4 +15,15 @@ AddOutputFilterByType DEFLATE text/html text/plain text/xml text/css application
 	ExpiresActive on
     ExpiresDefault "access plus 1 day"
 </FilesMatch>
+
+<FilesMatch "styles\.json$">
+	ExpiresActive on
+	ExpiresDefault "access plus 1 hour"
+</FilesMatch>
+
+<FilesMatch "\.html">
+	ExpiresActive on
+	ExpiresDefault "access plus 1 hour"
+</FilesMatch>
+
```

Comment:
Good catch, I'll fix that.

On Fri, 15 Apr 2016 20:05 Dan Stillman, <notifications@github.com> wrote:

> In htdocs/.htaccess
> <https://github.com/zotero/styles-repo/pull/16#discussion_r59925423>:
>
> > @@ -15,4 +15,15 @@ AddOutputFilterByType DEFLATE text/html text/plain text/xml text/css application
> >  	ExpiresActive on
> >      ExpiresDefault "access plus 1 day"
> >  </FilesMatch>
> > +
> > +<FilesMatch "styles\.json$">
> > +	ExpiresActive on
> > +	ExpiresDefault "access plus 1 hour"
> > +</FilesMatch>
> > +
> > +<FilesMatch "\.html">
> > +	ExpiresActive on
> > +	ExpiresDefault "access plus 1 hour"
> > +</FilesMatch>
> > +
>
> I don't think we want this, do we? Won't this prevent revalidation for an
> hour? If anything, we want this to be a few seconds, to avoid revalidation
> if people click through to View Source and back, but we want people to get
> the latest JSON as soon as it's ready. We just want them to get 304s in the
> meantime.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zotero/styles-repo/pull/16/files/c7c0f82ebc2d64facc056a0593985c9aae5de550#r59925423>
>